### PR TITLE
Update pt_BR.lang

### PR DESCRIPTION
**Pull request by ArukaAlter**

### CODE REVIEW

File: resources/assets/tconstruct/lang/pt_BR.lang [hunk 1]
```diff
@@ -396,12 +399,18 @@ item.tconstruct.arrow_head.name=Ponta de Flecha
...
 item.tconstruct.fletching.name=Pluma
 
+item.tconstruct.bolt_core.name=Haste de dardo
+
 # Material specific tools and toolparts
 # item.tconstruct.<tool>.<material>
 # item.tconstruct.pickaxe.wood=Wooden Pickaxe
 
 # Gadgets
 item.tconstruct.slime_boots.name=Botas de Slime
```
Comment (bonii-xx) [inline resources/assets/tconstruct/lang/pt_BR.lang]: Still missing green

Comment (ArukaAlter) [inline resources/assets/tconstruct/lang/pt_BR.lang]: Apparently I can't even ¯\_(ツ)_/¯

**Comment by bonii-xx** [comment]
You know you don't have to close PRs to make changes, right? :P

**Comment by ArukaAlter** [comment]
Well I'd be damned, but sometimes I just forget how Git works.

**Pull request closed without merging**